eppoclient/client.go

```diff
@@ -86,13 +86,21 @@ func (ec *EppoClient) GetStringAssignment(flagKey string, subjectKey string, sub
 }
 
 func (ec *EppoClient) GetJSONAssignment(flagKey string, subjectKey string, subjectAttributes Attributes, defaultValue interface{}) (interface{}, error) {
-	variation, err := ec.getAssignment(ec.configurationStore.getConfiguration(), flagKey, subjectKey, subjectAttributes, jsonVariation)
-	if err != nil || variation == nil {
+	variation, err := ec.getAssignment(ec.configurationStore.getConfiguration(), flagKey, subjectKey, subjectAttributes, jsonVariation, true)
+	if err != nil || variation.Raw == nil {
 		return defaultValue, err
 	}
 	return variation, err
 }
```

Comment:
❗ just flagging that `GetJSONAssignment` now really returns `AssignmentValue` instead of parsed value and this is a breaking change